packages/hardhat/deploy/00_deploy_your_contract.js

```diff
@@ -9,6 +9,7 @@ module.exports = async ({ getNamedAccounts, deployments }) => {
   const admins = [
     "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
     "0x34aA3F359A9D614239015126635CE7732c18fDF3",
+    "0xb010ca9Be09C382A9f31b79493bb232bCC319f01"
```

Comment:
For some reason event if I add my address here, I get the "admin only error funciton" when testing locally (contract deployed locally, and react app poiting to localhost)

![image](https://user-images.githubusercontent.com/2486142/144036990-4b7c22a0-bcd5-432b-b2bc-3969d710fa76.png)

Had to remove the adminOnly modifier to test this.

Could you test locally?

Reply:
Oh, I noticed that if I make this change in the contract, it works.

from:

```solidity
for (uint256 i = 0; i < admin.length; ++i) {
  _setupRole(DEFAULT_ADMIN_ROLE, admin[i]);
}
```

to

```solidity
for (uint256 i = 0; i < admin.length; ++i) {
  _setupRole(ADMINS_ROLE, admin[i]);
}
```

Note the DEFAULT_ADMIN_ROLE => ADMIN_ROLE change.

Reply:
So the Role access works like

DEFAULT_ADMIN => ADMIN_ROLE

Default admins can grant themselves the admin role, I’ll add some distinctions to make this more clear in the contract. They are also the users who can revoke ADMIN_ROLE

So we want to make sure the constructor still establishes default_admins. Fine for local testing thi

Reply:
Ok, I see. I was expecting to be an admin (with ADMIN_ROLE) when changing the deploy script and using my address. 

So I need to give myself the ADMIN_ROLE after the deploy, right? Maybe we call the grantRole function on the deploy script with some instructions to make it clearer.